src/modules/wallet/api/utils/getWalletType.ts

```diff
@@ -0,0 +1,16 @@
+import { GnosisSafeInfo, WalletType } from '../types'
+
+interface GetWalletTypeParams {
+  gnosisSafeInfo?: GnosisSafeInfo
```

Comment:
You can get just a boolean for this from https://github.com/cowprotocol/cowswap/blob/b1663a851d1c7a642e52810c3d8c399420e6a479/src/modules/wallet/web3-react/hooks/useWalletMetadata.ts#L104

Reply:
This function is not a hook. I've essentially just extracted the existing logic for this.

Reply:
Ah - makes sense. In this case this one is only used for an updater which already has this information though.